packages/botonic-plugin-dialogflow/src/index.js

```diff
@@ -3,10 +3,16 @@ import axios from 'axios'
 export default class BotonicPluginDialogflow {
   constructor(options) {
     this.options = options
+    this.defaultVersion = '20150910'
   }
 
-  async query(data, lang, sessionId) {
-    // Data, lang and sessionId are required parameters
+  async query(data, lang, sessionId, version = this.defaultVersion) {
+    // Data, lang, sessionId and version are required parameters
+    // See https://dialogflow.com/docs/reference/agent/query for more information
+
+    if (this.options.version) {
+      version = this.options.version
+    }
     return await axios({
       headers: {
         Authorization: 'Bearer ' + this.options.token
```

Comment:
Try to use template strings with backtick \` as much as you can. In this case: `` `Bearer ${this.options.token}` ``